.github/workflows/pr-title.yaml

```diff
@@ -0,0 +1,25 @@
+name: PR Title Linter
+# Prevent writing to the repository using the CI token.
+# Ref: https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#permissions
+permissions: read-all
+on:
+  pull_request:
+    # By default, a workflow only runs when a pull_request's activity type is opened,
+    # synchronize, or reopened. We explicity override here so that PR titles are
+    # re-linted when the PR text content is edited.
+    types:
+      - opened
+      - edited
+      - reopened
+      - synchronize
+jobs:
+  pr-lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: morrisoncole/pr-lint-action@v1.7.0
+        with:
+          title-regex: "^[A-Z].*[^.]$"
```

Comment:
Should we also forbid other terminal punctuation, like question marks or exclamation points?

Reply:
Yea sgtm, I'm not crazy good in regex land though, so once we get this in feel free to adjust (or just push here)

Reply:
Sure, can do. Might be nice to wait on copy-pasting this to every repo until we do this - otherwise it's twice as many PRs to ship and review. 

This should be a pretty trivial fix - add \?\! to the exclude set.

Reply:
Apparently @rubensf has some magic script that can do these types of rollouts widely, so @rubensf - coordinate with @akshayjshah once we have the final regex

Reply:
Also the exclude set as it is now just bans `.` at the end of the PR title, we likely want to allow `.` in the middle though